src/formInputs/DatePicker/index.tsx

```diff
@@ -305,7 +305,7 @@ export class DatePicker extends PureComponent<DatePickerProps, DatePickerState>
         if (
           (minDate && minDate.getTime() > firstDate.getTime()) ||
           (maxDate && maxDate.getTime() < secondDate.getTime()) ||
```

Comment:
```suggestion
     if (minDate && minDate.getTime() > firstDate.getTime()) {
       firstDate = minDate;
     }
     if (maxDate && maxDate.getTime() < secondDate.getTime()) {
       secondDate = maxDate;
     }
     if (
```
이 코드의 용도는 minDate보다 입력값이 작을 때 minDate를, maxDate보다 입력값이 클 때 maxDate값으로 대체해서 넣는거라서 이런식으로 가는게 조금 더 깔끔할 것 같아요!